### DISCUSSION TITLE
Dialogs & Bottomsheets

### DISCUSSION THREAD

**Discussion by hrach** [bumble-tech/appyx#77] [Q&A]
Hi, I haven't found any mention of the dialog/bottom-sheet support. Is there a plan for it?

**Comment by KovalevAndrey** [comment]
Hi @hrach. I think you can use [bottom-sheets](https://foso.github.io/Jetpack-Compose-Playground/material/modalbottomsheetlayout/) and [dialogs](https://foso.github.io/Jetpack-Compose-Playground/material/alertdialog/) directly inside @View method in the node

**Reply by hrach** [reply]
I don't expect it works/it doesn't work:
- just tried: the dialogs scrim is not transparent and doesn't show the previous destination underneath
- even if it would show, there would not be properly "alive", as it would be stopped (current backstack impl)
- some animation coordination is needed as swiping bottomsheet down should already animate somehow and no further nav-stack animation should bedone

**Comment by arok** [comment]
Any news or samples on this topic?

**Comment by enikebraimoh** [comment]
Anything yet?

**Reply by KovalevAndrey** [reply]
hey guys, sorry for the delayed response. We have an example in `:navmodel-samples` how one can come up with Modal `NavModel`. In the meantime, we're working on the gesture driven NavModel which should support the dragging use case

**Comment by GMKornilov** [comment]
If you want to treat bottom sheet as NavTarget(and/or show bottom sheet above all app's content), I think you can use approach similar to Accompanist Navigation Material - place ModalBottomSheetLayout in RootNode and create separate NavModel for managing sheet state

Created small sample for this case - https://github.com/GMKornilov/appyx-bottom-sheet-sample

**Comment by SeleznevAM** [comment]
Hello colleagues. Can you update bottomsheet sample to new BottomSheetLayout pls
